src/parseMachine.ts

```diff
@@ -54,6 +54,10 @@ export function parseMachines(sourceJs: string): Array<StateNode> {
       log: console.log,
       warn: console.warn,
     },
+    setInterval,
+    setTimeout,
+    clearTimeout,
+    clearInterval,
```

Comment:
Have you tested this locally though? :troll:

Reply:
Ooo, good point

Reply:
@Andarist Ready for re-review!